### DISCUSSION TITLE
Can't login Wiki.js after configuring the wrong URL

### DISCUSSION THREAD

**Discussion by pasa2006** [requarks/wiki#6811] [Help / Questions] [answered]
Hi,

I completed the installation of Wiki.js on a Windows server; however, I entered the wrong URL in the "Site URL" section of the web page. I failed to change the protocol from HTTPS to HTTP since we haven't configured HTTPS yet and didn't include the default port number (i.e., 3000) in the URL.

I cannot reach to Wiki.js server now to continue the configuration.

Is there a way for me to fix this misconfiguration issue?

Thanks,

**Comment by jaypoc** [comment, marked as answer]
Just made a changed based on this comment and did the same thing to my own wiki. I was able to resolve by editing the configuration in the database itself. Look in the **settings** table.

**Reply by pasa2006** [reply]
Thank you!
